src/models/ProxyHits.js

```diff
@@ -0,0 +1,52 @@
+const Joi = require('joi');
+const BaseModel = require('./BaseModel');
+
+const schema = {
+	id: Joi.number().integer().min(1).required().allow(null),
```

Comment:
Remove and use (`proxyId`, `date`) as the primary key (everything should work like in `FileHits`).